src/main/minio.js

```diff
@@ -2013,6 +2013,12 @@ export class Client {
   }
 }
 
+// Promisify various public-facing APIs on the Client module.
+Client.prototype.makeBucket = promisify(Client.prototype.makeBucket)
+Client.prototype.listBuckets = promisify(Client.prototype.listBuckets)
+Client.prototype.bucketExists = promisify(Client.prototype.bucketExists)
+Client.prototype.removeBucket = promisify(Client.prototype.removeBucket)
```

Comment:
Yes, it works right now (with transpiling) and re-assigning class functions works in the REPL (native), if we ever stop transpiling.